mobius-android/src/main/java/com/spotify/mobius/android/MobiusAndroidViewModel.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * -\-\-
+ * Mobius
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.mobius.android;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import com.spotify.mobius.First;
+import com.spotify.mobius.Init;
+import com.spotify.mobius.MobiusLoop;
+import com.spotify.mobius.MobiusLoop.Factory;
```

Comment:
nit: I think the code is more readable if this isn't statically imported - the `MobiusLoop.` part of the name is useful to see which kind of factory it is.

Reply:
👍